inductiva/fluids/scenarios/wind_tunnel/README.md

```diff
@@ -0,0 +1,151 @@
+
+Now, the user is ready to simulate the steady state. Here, the user chooses the object to be inserted inside the wind tunnel. This object is defined with a geometry file in STL or OBJ format (currently tested). Notice, that appropriate meshing will be done during the simulation.
+
+The user can further define the following simulation parameters:
+- simulator: The simulator to be used for the simulation. Currently, only `OpenFOAM` is supported.
+- num_iterations: Number of iterations for the steady-state simulation.
+- resolution: Resolution of the meshing. The higher the resolution, the finer the meshing.
+
+Moreover, the hardware and interaction are configured with the usual general parameters - `machine_group`, `run_async`, `n_cores`.
+Launching a simulation returns a task object, which can be used to verify the status of the simulation, get the simulation outputs and access instantly post-processing tools. See more in [Tasks](inductiva/tasks/README.md).
+
+### Example:
+
+```python
+import inductiva
+
+# Initialize the scenario
+scenario = inductiva.fluids.scenarios.WindTunnel(
+    flow_velocity=[30, 0, 0],
+    dimensions={"x": [-5, 15], "y": [-5, 5], "z": [0, 8]})
+
+# Run a simulation
+task = scenario.simulate(
+    object_path="f1.obj",
+    num_iterations=1000, resolution=0.5,
+    run_async=True, n_cores=4)
+
+# Get the simulation output on your local machine.
+output = task.get_output()
+```
+
+## Output and Post-Processing:
+
+The `WindTunnel` scenario allows users to extract metrics about the object under the established flow velocity.
+The simulation outputs of the wind tunnel are:
+- `pressure_field`: Pressure field of the airflow over the object.
+- `streamlines`: Streamlines of the airflow over the domain and interacting with the object.
+- `flow_slice`: Slice of the flow that represents the airflow over the domain and interacting with the object.
+- `force_coefficients`: Force coefficients that represent the forces acting on the object. These are the drag and lift coefficients.
+
+### Default vs Full Post-processing Tools
+
+Since at times the output files can be large, the user can choose to get only some of the default outputs that are post-processed
+on run-time by just doing `task.get_output()` (e.g., pressure field, streamlines and flow_slice). Otherwise, the user can choose to get all the outputs with `task.get_output(full=True)` and post-process as he wishes locally. 
+
+In any case, to obtain the metrics above the user can use the following post-processing tools:
+- `get_output_mesh`: Returns the mesh of the airflow over the entire domain and data on the object.
+- `get_object_pressure_field`: Returns the pressure field of the airflow over the object.
+- `get_streamlines`: Input parameters - `max_time`, `n_points`, `initial_step_length`, `source_radius`, `source_center`. Returns a mesh of the streamlines with pressure and velocity components of airflow over the domain.
```

Comment:
If you mention the input parameters, you need to explain what each of them is.

Reply:
I feel that here that will be to heavy and the argument names are very explicit for those interested in doing post-processing. I think in the future, with the read the docs, that will be simpler to do.